src/test/java/org/jenkinsci/plugins/registry/notification/CoordinatorTest.java

```diff
@@ -73,10 +73,12 @@ public void testTwoTriggered() throws Exception {
         FreeStyleProject one = j.createFreeStyleProject();
         one.addTrigger(new DockerHubTrigger(new TriggerOnSpecifiedImageNames(Arrays.asList("csanchez/jenkins-swarm-slave"))));
         one.getBuildersList().add(new MockBuilder(Result.SUCCESS));
+        one.setQuietPeriod(0);
```

Comment:
not needed - but schedules the jobs immediatly and saves a few seconds of the test run :)